client/src/Containers/Addtitle.jsx

```diff
@@ -1,59 +1,119 @@
-import { Button, Grid, Paper, TextField, Typography } from '@material-ui/core';
-import React from 'react';
-import useStyles from './Styles/AddQuestionStyles';
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, TextField } from '@material-ui/core';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
 
 
 
-const Addquestion = () => {
-  const classes = useStyles();
+const Addtitle = ({ open , setOpen , data , setData }) => {
+  const [quizLink, setQuizLink] = useState('');
+  const [openQuizLink, setOpenQuizLink] = useState(false);
+  
+
+  function sendToDB()
+  {
+    const title = document.getElementById('title-input').value; // Get the title
+    setData( dataObj => {
+      const obj = dataObj;
+      obj.title = title;
+      
+      return obj;
+    }); // Adding the title to the object to be sent
+
+    axios.post( 'http://localhost:5000/addquiz' , data)

```

Comment:
Explain the axios post with proper comments like what this post request is doing?